tools/third_party_modified/mozlog/mozlog/formatters/grouping.py

```diff
@@ -163,8 +164,28 @@ def wrap_and_indent_lines(self, lines, indent):
             output += indent + u"\u2514 %s\n" % lines[-1]
         return output
 
+    def get_lines_for_unexpected_subtest_result(
```

Comment:
Rather than duplicating what was `get_lines_for_unexpected_result`, how about make the first argument the "formatted test key" (i.e., it becomes the caller's responsibility to pass either `subtest_name` or `self.get_test_name_output(subsuite, test_name)`)?

Reply:
Done, thanks